spring-cloud-generator/src/test/java/com/google/cloud/generator/spring/composer/goldens/EchoSpringAutoConfigurationFull.golden

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2024 Google LLC
+ * Copyright 2025 Google LLC
```

Comment:
hm, this is already up to 2025 in [main branch](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/d9380203461189626b7459f930f9300bdb236744/spring-cloud-generator/src/test/java/com/google/cloud/generator/spring/composer/goldens/EchoSpringAutoConfigurationFull.golden#L2). And I notice this change is not in #3411. Can you merge main into this?